scripts/benchmark.js

```diff
@@ -14,126 +14,181 @@ const fs     = require("fs");
 const path   = require("path");
 const compile = require("google-closure-compiler-js").compile;
 
-const filename = process.argv[2];
-if (!filename) {
-  console.error("Error: No filename specified");
-  process.exit(1);
+const packagename = process.argv[2];
+var filename = process.argv[3];
+
+if (!packagename) {
+	console.error("Error: No package specified");
+	process.exit(1);
 }
 
+console.log("package: " + packagename);
+
+const pathToScripts = "./scripts";
```

Comment:
I still like the idea of assigning the value to pathToScripts just to make it clear where things are being installed and whatnot